pkg/operator/ceph/controller/spec.go

```diff
@@ -456,3 +463,52 @@ func StoredLogAndCrashVolumeMount(varLogCephDir, varLibCephCrashDir string) []v1
 		},
 	}
 }
+
+// GenerateLivenessProbeExecDaemon makes sure a daemon has a socket and that it can be called and returns 0
+func GenerateLivenessProbeExecDaemon(daemonType, daemonID string) *v1.Probe {
+	confDaemon := getDaemonConfig(daemonType, daemonID)
+
+	return &v1.Probe{
+		Handler: v1.Handler{
+			Exec: &v1.ExecAction{
+				// Run with env -i to clean env variables in the exec context
+				// This avoids conflict with the CEPH_ARGS env
```

Comment:
Why is CEPH_ARGS set in the container? Should we remove that from the base container?

Reply:
It is set on the OSD container to connect to the mons and will conflict with that command.